GameData/ROTanks/Compatibility/TUColorPresets.cfg

```diff
@@ -0,0 +1,42 @@
+//------------------------- colors -------------------------
+KSP_COLOR_PRESET
+{
+	name = atlasvccb
+	title = Atlas V CCB
+	color = 192, 128, 99
+	specular = 60
+	metallic = 30
+}
+KSP_COLOR_PRESET
+{
+	name = atlasvccbfuelline
+	title = Atlas V CCB Fuel Line
+	color = 192, 128, 99
+	specular = 60
+	metallic = 30
+}
+KSP_COLOR_PRESET
+{
+	name = centaurcommon
+	title = Centaur Common Tank
+	color = 239, 239, 239
+	specular = 30
+	metallic = 30
+}
+KSP_COLOR_PRESET
+{
+	name = centaurcommonmount
+	title = Centaur Common Mount
+	color = 236, 218, 176
+	specular = 30
+	metallic = 30
+}
+//------------------------- color groups -------------------------
+PRESET_COLOR_GROUP
+{
+	name = Historical Tanks
+	color = atlasvccb
+	color = atlasvccbfuelline
+	color = centaurcommon
+	color = centaurcommonmount
+}
```

Comment:
Does this color group need to be added to anything?
Are things happy with a space in the _name_ field, versus a title?

Reply:
no it works fine